pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -610,7 +612,9 @@ var _ = Describe("Manager", func() {
 			vmi := newVMI(testNamespace, testVmName)
 
 			mockConn.EXPECT().QemuAgentCommand(`{"execute":"`+string(agentpoller.GetFSFreezeStatus)+`"}`, testDomainName).Return(expectedFrozenOutput, nil)
-			mockConn.EXPECT().QemuAgentCommand(`{"execute":"guest-fsfreeze-thaw"}`, testDomainName).Return("1", nil)
+			mockConn.EXPECT().LookupDomainByName(testDomainName).DoAndReturn(mockDomainWithFreeExpectation)
+			mockDomain.EXPECT().FSThaw(nil, uint32(0)).Times(1)
```

Comment:
Shouldnt you expect here also:
mockDomain.EXPECT().Free().Times(1)

Reply:
Same here. The reason that `Free().AnyTimes()` is expected in another test, because it's called multiple times.